### ISSUE TITLE
File name not always shown, can be misleading

### ISSUE THREAD

**Issue by pedro-w** [jonase/kibit#205]
I've got a project which uses reader macros, which aren't supported. This causes kibit to print a message that it is skipping the rest of the file. Unfortunately if that's the only problem it doesn't say which file. 

Example
```
At src/widum_server/device.clj:338:
Consider using:
  (log/fatal "Error in address or port")
instead of:
  (do (log/fatal "Error in address or port"))
Check failed -- skipping rest of file
[line 447, col 15] No reader function for tag widum/CS.
Check failed -- skipping rest of file
[line 302, col 40] No reader function for tag widum/CS.
```
Here there is advice for `device.clj` followed by kibit stopping after line 447 but also a problem with a different file on line 302.

It would be great if it could name the file at this point.

Using kibit 0.1.6, clojure 1.9.0-beta

**Comment by bago2k4** [comment]
I have this issue too, here is my output:
```
$ lein kibit src/oc/web/components/ui/
Check failed -- skipping rest of file
[line 25, col 101] Unmatched delimiter }.
Check failed -- skipping rest of file
[line 125, col 52] No reader function for tag js.
Check failed -- skipping rest of file
[line 29, col 45] No reader function for tag js.
Check failed -- skipping rest of file
[line 62, col 8] No reader function for tag js.
Check failed -- skipping rest of file
[line 16, col 22] No reader function for tag js.
```
It's a cljs project that uses boot but i had  create a temporary project.clj and use `lein kibit path/to/folder` to track down the issue.
Both `boot check/check-kibit` and `lein kibit` didn't print the filename so i guess it's a problem of this lib itself.

**Comment by pedro-w** [comment]
This issue was fixed by pull request #212
